Scripts/s4ap/utils/s4ap_sim_utils.py

```
@@ -0,0 +1,70 @@
+from sims.sim import Sim
+from sims.sim_info import SimInfo
+from sims.sim_info_base_wrapper import SimInfoBaseWrapper
+from sims.sim_info_manager import SimInfoManager
+
+class S4APSimUtils:
+
+    @staticmethod
+    def get_sim_first_name(sim_info: SimInfo):
+        if sim_info is None or not hasattr('first_name'):
+            return ''
+        return getattr(sim_info, 'first_name')
```

Ruff (0.14.7)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)